Jenkinsfile

```diff
@@ -31,3 +42,37 @@ node('linux') {
 			sh 'make test'
 	}
 }
+
+def git_commit = ''
+def go_list = ''
+
+stage 'Go & Git Info'
+node('linux') {
+
+	def gopath = gopath_linux()
+	def workdir = workdir_linux(gopath)
+
+	dir(workdir) {
+		checkout scm
+
+		// HACK: Recommended approach for getting command output is writing to and then reading a file.
+		sh 'mkdir -p tmp'
+		sh 'go list . > tmp/GO_LIST'
```

Comment:
@arschles we are oh so close.  The only thing we need to resolve is how to get the `github.com/<user>/controller-sdk-go` string.  As it stands, I believe `go list .` will always return `github.com/deis/controller-sdk-go`, yet the standard approach is to PR a change in this repo from a fork.  

Ideas?